action.yml

```diff
@@ -1,11 +1,7 @@
-name: 'run-script'
+name: 'octorelease-script'
```

Comment:
breaking change incoming! 😱

Reply:
Not sure if this is a breaking change, Idk where the `name` property gets used unless you publish your action to the marketplace 😛 There are definitely breaking changes to the way scripts behave in this PR though - going to open a PR to update Imperative workflows ASAP after merging this.